src/test/java/com/imsweb/geocoder/GeocodeOutputTest.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) 2017 Information Management Services, Inc.
+ */
+package com.imsweb.geocoder;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import junitx.util.PrivateAccessor;
+
+public class GeocodeOutputTest {
+
+    // The important behavior in the GeocodeOutput class is tested in GeocoderTest, but wanted a test for some of the helper methods
+    @Test
+    public void testIntValue() throws Throwable {
+        Class<?>[] clazz = {String.class};
+
+        Object[] param = {"7"};
+        Object result = PrivateAccessor.invoke(GeocodeOutput.class, "intValue", clazz, param);
```

Comment:
As an FYI for the future, I've never been a fan of testing private methods in this way.  I think its good you tested the method, put I've always found the path of least resistance is to make the method package private.  We can leave this for now.